.pre-commit-config.yaml

```diff
@@ -57,11 +69,9 @@ repos:
         stages: [commit-msg]
 
   - repo: https://github.com/thclark/pre-commit-sphinx
-    rev: 0.0.1
+    rev: 0.0.3
     hooks:
       - id: build-docs
         language_version: python3
-        # Use dev dependencies to install sphinx and related libraries
-        # as well as installing the root module (required for autodoc)
         additional_dependencies:
-          - "django_twined@file://localhost/workspace#egg=django_twined"
```

Comment:
Yeah I could see what it was for, I only changed it because it doesn't work if you're not using VSCode (presumably in a devcontainer) - I'll look into an agnostic way to keep it functioning like that

Reply:
was it not working for you before?

Reply:
The thing is, I spent several hours figuring out how to do this so it'd flawlessly build the latest docs changes with autodoc on the API and I'm pretty sure that `.` is the first thing I tried...

Reply:
It wasn't, I wouldn't have changed it otherwise 😛

Reply:
I'm looking at the deployed docs and I don't see the API documented at all currently

Reply:
Yeah I think I copied this config from django-gcp which I have set that up for already. It'll come, we need to do a hell of a lot of docs work.